README.md

```diff
@@ -74,41 +86,160 @@ When logging in:
 3. Connect your wallet and approve the authentication
 4. CLI will automatically detect successful authentication
 
-### Building Projects
+### Configuration
 
-Build your assertion contracts:
+Manage your PCL configuration:
 
 ```bash
-pcl build [OPTIONS] [ASSERTIONS]...
+pcl config [COMMAND]
 
-Arguments:
-  [ASSERTIONS]...  Names of assertion contracts to build
-
-Options:
-  -h, --help  Print help
+Commands:
+  show    Display the current configuration
+  delete  Delete the current configuration
 ```
 
-### Phorge Commands
+Configuration is stored in `~/.pcl/config.toml` and includes:
 
-Phorge is a Foundry-compatible development environment for assertions:
+- Authentication token
+- Pending assertions for submission
+- Project settings
 
-```bash
-pcl phorge [OPTIONS] [ARGS]...
+### Testing
 
-Arguments:
-  [ARGS]...  Arguments to pass to the phorge command
+Run tests using Phorge (a Forge-compatible development environment). It's a minimal fork of forge to support out assertion execution cheatcodes, so `pcl test` behaves identically to `forge test`.
 
-Options:
-  -r, --root <ROOT_DIR>              The root directory for the project.
-  -a, --assertions <ASSERTIONS_DIR>  The directory containing assertions 'src' and 'test' directories.
-  -h, --help  Print help
-```
+```pcl test -h
+Run tests using Phorge
 
-Common phorge subcommands:
+Usage: pcl test [OPTIONS] [PATH]
 
-```bash
-pcl phorge test      # Run tests
-pcl phorge script    # Run scripts
+Options:
```

Comment:
nit: I don't think we need to paste the entire output of this command since it's the same as forge test, so maybe just mention that and avoid adding 100+ redundant lines to the readme

Reply:
[ed43a5d](https://github.com/phylaxsystems/pcl/pull/44/commits/ed43a5dd794c6694d6dbc1643162fcb63fab458d)